backend/app/services/implementations/log_records_service.py

```diff
@@ -24,7 +24,6 @@ def __init__(self, logger):
 
```

Comment:
For reference these changes are extras Owen found in the create:
- datetime now added in req
- note required